### ISSUE TITLE
How to output the ternarized weight?

### ISSUE THREAD

**Issue by manwu1994** [elliothe/Ternarized_Neural_Network#1]
Hello.
You used a pre-trained model as the initial model for initialization weight. 
Does the following code perform the steps mentioned above?

```
def resnet101b_fq_lq_tex1(num_classes=1000):
    model = ResNet(Bottleneck, [3, 4, 23, 3], fp_fl=False, fp_ll=False)
    model.load_state_dict(model_zoo.load_url(model_urls['resnet101']))
    return model
```

Additionally, the output weights are still full-precision? how to get (-w,0,w) and how to output the scaling factor _a_ of the (-w,w)?
```
class quanConv2d(nn.Conv2d):
    #def __init__(self,*args,**kwargs):
        #super(quanConv2d,self).__init__(*args,**kwargs)
    def forward(self, input):
        tfactor_list = [0.05]
        weight = _quanFunc(self.weight) #(tfactor=tfactor_list[0])
        output = F.conv2d(input, weight, self.bias, self.stride, self.padding, self.dilation, self.groups)
        
        return output 
```
If try to use self. weight to replace  _quanFunc(self. weight), the accuracy will decrease drastically
Thank you a lot for all.
